### PR TITLE
[IMP] mail: show footer separator correctly

### PR DESCRIPTION
**Pull request by fbarbe00**
Purpose
=======
Avoid displaying "|" if there is nothing to separate in footer of email
template for sale order.

Task-2798501

### CODE REVIEW

File: addons/mail/data/mail_templates_email_layouts.xml [hunk 1]
```diff
@@ -110,14 +110,14 @@
...
-                </t>
-                <t t-if="company.website"> |
-                    <a t-att-href="'%s' % company.website" style="text-decoration:none; color: #999999;">
-                        <t t-esc="company.website"/>
-                    </a>
-                </t>
+                <t t-if="company.email and company.phone"> |</t>
+                <a t-if="company.email" t-att-href="'mailto:%s' % company.email" style="text-decoration:none; color: #999999;">
+                    <t t-esc="company.email"/>
+                </a>
```
Comment (std-odoo) [inline addons/mail/data/mail_templates_email_layouts.xml:116]: while you change this, the `t-esc` can be included in the `<a/>`

File: addons/mail/data/mail_templates_email_layouts.xml [hunk 2]
```diff
@@ -110,14 +110,14 @@
...
+                <t t-if="company.website and (company.phone or company.email)"> |</t>
+                <a t-if="company.website" t-att-href="'%s' % company.website" style="text-decoration:none; color: #999999;">
+                    <t t-esc="company.website"/>
+                </a>
```
Comment (std-odoo) [inline addons/mail/data/mail_templates_email_layouts.xml:120]: same here

File: addons/mail/data/mail_templates_email_layouts.xml [hunk 3]
```diff
@@ -211,14 +211,14 @@
...
-                </t>
-                <t t-if="company.website"> |
-                    <a t-att-href="'%s' % company.website" style="text-decoration:none; color: #999999;">
-                        <t t-esc="company.website"/>
-                    </a>
-                </t>
+                <t t-if="company.email and company.phone"> |</t>
+                <a t-if="company.email" t-att-href="'mailto:%s' % company.email" style="text-decoration:none; color: #999999;">
+                    <t t-esc="company.email"/>
+                </a>
```
Comment (std-odoo) [inline addons/mail/data/mail_templates_email_layouts.xml:217]: same here

File: addons/mail/data/mail_templates_email_layouts.xml [hunk 4]
```diff
@@ -211,14 +211,14 @@
...
+                <t t-if="company.website and (company.phone or company.email)"> |</t>
+                <a t-if="company.website" t-att-href="'%s' % company.website" style="text-decoration:none; color: #999999;">
+                    <t t-esc="company.website"/>
+                </a>
```
Comment (std-odoo) [inline addons/mail/data/mail_templates_email_layouts.xml:221]: same here

File: addons/mail/data/mail_templates_email_layouts.xml [hunk 5]
```diff
@@ -18,13 +18,13 @@
         <a t-if="has_button_access"
                 t-att-href="button_access['url']"
                 style="padding: 8px 12px; font-size: 12px; color: #FFFFFF; text-decoration: none !important; font-weight: 400; background-color: #875A7B; border: 0px solid #875A7B; border-radius:3px">
-            <t t-esc="button_access['title']"/>
+            <t t-out="button_access['title']"/>
```
Comment (std-odoo) [inline addons/mail/data/mail_templates_email_layouts.xml:21]: in your commit message, no need to write the filepath

Maybe just something like this ?
> Replace t-esc with t-out in email templates layout

**Review by std-odoo — COMMENTED** [review]
@fbarbe00 Hi :) 

Some minor comments, otherwise LGTM :)

**Comment by tde-banana-odoo** [comment]
@robodoo r+ rebase-ff

**Pull request closed without merging**